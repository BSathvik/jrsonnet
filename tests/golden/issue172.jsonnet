[x.value for x in std.objectKeysValues({ a: b })]
