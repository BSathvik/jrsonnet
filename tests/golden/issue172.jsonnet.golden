variable is not defined: b
    issue172.jsonnet:1:45-47: variable <b> access
    issue172.jsonnet:1:4-10:  field <value> access
    elem <0> evaluation